src/main/java/com/facebook/openwifirrm/modules/Modeler.java

```diff
@@ -83,7 +83,14 @@ public static class DataModel {
 		// At minimum, we may want to aggregate recent wifi scan responses and
 		// keep a rolling average for stats.
 
-		/** List of latest wifi scan results per device. */
+		/**
+		 * The "result" of a wifiscan can include multiple responses.
+		 * {@code latestWifiScans} maps from an AP (serial number) to a list of most
```

Comment:
nit: don't put @ reference to self (the thing you're already commenting on)

Reply:
I will change this.